### PR TITLE
Various css changes to colors, fonts, margins

### PR DESCRIPTION
**Pull request by roberthouse54**
#### What are the relevant tickets?

These are general style changes not necessarily for any ticket. I made changes to styles in the dashboard, the user page, and the learner search.

#### Where should the reviewer start?

Look at the changes to the css and js files and see that nothing looks broken.

#### Any background context you want to provide?

This is an ongoing process of tweaking css styles that will continue.

### CODE REVIEW

File: static/js/components/UserChip.js [hunk 1]
```diff
@@ -18,7 +18,6 @@ const UserChip = ({ profile }: {profile: Profile}): React$Element<*> => (
         { mstr(getEmployer(profile)) }
       </span>
       <a href={`/users/${profile.username}`} className="mm-minor-action">
-        <Icon name="person" />
```
Comment (noisecapella) [inline static/js/components/UserChip.js]: Can you remove line 4 in this file since we no longer need `Icon` after your change?

Comment (amir-qayyum-khan) [inline static/js/components/UserChip.js]:
```log
/src/static/js/components/UserChip.js
  4:8  error  'Icon' is defined but never used  no-unused-vars\
```

**Comment by amir-qayyum-khan** [comment]
👍 💯 🎉

**Comment by noisecapella** [comment]
Why was the `Icon` change reverted?

**Comment by roberthouse54** [comment]
@noisecapella I made a mistake deleting it and asked Amir to revert the change

**Pull request merged**